webpack/assets/javascripts/hosts/tableCheckboxes.js

```diff
@@ -229,9 +246,12 @@ export function buildRedirect(url) {
 }
 
 function paginationMetaData() {
-  const pagination = $('#pagination');
-  const total = pagination.data('count');
-  const perPage = $('#per_page').val();
+  const total = Number(
+    document.getElementsByClassName('pagination-pf-items-total')[0].textContent
```

Comment:
This pagination is a react component, what about  connecting it to redux and  getting this data directly from the store? Looks better then using classes which can be break by patternfly's implementation.

Reply:
This fix is pretty urgent if i'm not wrong @ohadlevy 
so I worked with what we have. eventually with the router this whole file will be re-written